scripts/download.sh

```diff
@@ -52,24 +52,15 @@ mkdir -p beta-amd64/
 tar xzvf beta-amd64.tar.gz -C beta-amd64/
 chmod +x beta-amd64/buildkite-agent
 
+rm -rf agent-master/
+download https://github.com/buildkite/agent/archive/master.zip master.zip
+unzip master.zip
+cd agent-master/
+docker build --tag agent-master .
+docker run --rm -v "$PWD:/go/src/github.com/buildkite/agent" agent-master scripts/utils/build-binary.sh "linux" "amd64" "edge"
+cd ../
```

Comment:
We don't push edge releases to Github… and you can't install an edge release from the installer scripts. The only place they're pushed to is the experimental apt, rpm, etc.

We should probably push them to a "binaries + sha" dump http index somewhere… like download.buildkite.com, and not just apt/rpm etc. But we don't do that currently. So this is probably the best bet for now?